koku/masu/util/azure/common.py

```diff
@@ -61,12 +59,9 @@
     "EffectivePrice",
     "CostInBillingCurrency",
     "ConsumedService",
-    "ResourceId",
-    "OfferId",
     "AdditionalInfo",
     "ServiceInfo1",
     "ServiceInfo2",
-    "ResourceName",
```

Comment:
Why are we removing some of the columns from here?

Reply:
these can go back now :+1: thanks